extensions/interactions/InteractiveMap/InteractiveMap.js

```diff
@@ -108,3 +110,49 @@ oppia.directive('oppiaShortResponseInteractiveMap', [
     };
   }
 ]);
+
+oppia.factory('interactiveMapRulesService', function() {
+  var RADIUS_OF_EARTH = 6371.0;
+  var _degreesToRadians = function(angle) {
```

Comment:
Remove leading underscores -- as part of the frontend refactor we are getting rid of these. Methods that aren't in the returned object are considered 'private'.